cookbook/ui_customization/create_a_new_field_type.rst

```diff
@@ -0,0 +1,146 @@
+How to add a new field type
+===========================
+
+If you want a custom rendering for one of your attributes, attribute type or reference data you need to create a new field type. In this cookbook, we will go through each step needed to create a custom field type.
+
+Let's say that we want to have a slider to represent each "number attribute" that have a minimum and a maximum value limit.
```

Comment:
"Let's say that we want" -> "Let's say we want"
"that have a minimum" -> "having a minimum"